syntax = "proto3";

package tensorflow.quantization;

import "tensorflow/core/framework/graph.proto";

option cc_enable_arenas = true;

// Represents an exported TensorFlow model. It consists of a GraphDef and extra
// metadata required for building a SavedModel. This message is primarily used
// to "export" the model produced from various quantization passes in c++ to
// Python layer.
message ExportedModel {
  GraphDef graph_def = 1;

  // Name of the initialization node (TF Operation) used for initializing
  // resources like hash tables upon loading.
  string init_node_name = 2;
}
